services/nginz/Makefile

```diff
@@ -69,8 +69,11 @@ $(DIST):
 BUNDLE=nginx-$(VERSION).tar.gz
 
 src: $(TMPDIR)/$(BUNDLE)
+	#Find keys on https://nginx.org/en/pgp_keys.html
+	gpg --verify $(TMPDIR)/$(BUNDLE).asc
```

Comment:
Since the signature is detached you also need to provide `$(TMPDIR)/$(BUNDLE)` as a second argument.